### ISSUE TITLE
require-system-font-fallback: false positive when value is a single var()

### ISSUE THREAD

**Issue by azat-io** [yuschick/stylelint-plugin-defensive-css#205]
**Describe the bug**

There is an issue when using the `defensive-css/require-system-font-fallback` rule.

The rule flags `font` / `font-family` declarations whose value is a single `var(...)`, even when the variable resolves to a value that includes a proper fallback. Since the fallback is centralized in one custom property, every consumer is reported as a false positive.

**To Reproduce**

```css
:root {
  --font-family-base: 'Co Text', sans-serif; /* fallback is here */
  --font-m: 400 1rem / 1.5 var(--font-family-base);
}

.title {
  font: var(--font-m); /* ✖ reported, but fallback exists via the variable chain */
}
```

**Expected behavior**

When a declaration value is composed entirely of `var(...)` references, the rule should skip it.

**Comment by yuschick** [comment]
Thanks for reporting. Yes, this is a false positive, however, for a slightly different reason.

The plugin does not resolve `var()` values. So `var(--font-m)` should be skipped because it's a `var` and not because it resolves to a value with a fallback defined.

In the short term, you can use the `ignore` option to bypass this use case, but I will look into getting the `var` checks to correctly skip.

**Comment by yuschick** [comment]
Addressed in #208 and will be part of the next minor release.
